third_party/blink/renderer/core/page/context_menu_controller.cc

```diff
@@ -403,12 +403,16 @@ static gfx::Rect ComputeSelectionRect(LocalFrame* selected_frame) {
 
 bool ContextMenuController::ShouldShowContextMenuFromTouch(
```

Comment:
I see this method is called from 
`ContextMenuController::ShowContextMenu()` -- could we just
not instantiate this `ContextMenuController` class? IIUC
we shouldn't have any context menu whatsoever, so why have 
it at all?